### PR TITLE
Placeholder address update

### PR DESCRIPTION
**Pull request by zone117x**
Update to a newly received placeholder address.

### CODE REVIEW

File: blockstack/blockstackd.py [hunk 1]
```diff
@@ -3804,8 +3804,8 @@ class Program {
             'not_distributed_88a422b7ccaa54e5d8225d5130dc562ffdc57cc54bf274f2b631061ae86d0dc0': 'SM1TP7CNY63KQY7DVRDVPNE1X73ND3PT8JX46971V',
             'not_distributed_8941f1658fc78107ad34cf4d8e96a920a5a3257384188fb21e701c5e6a4b81fb': 'SM2ZP9PPEHXGPAAMYRJNMPVET8WRKBSYY1AZ8SWHP',
             'not_distributed_d6d337469b6283046ed909a03a1f45d862d6c1361d5d0ee56ae660e25866b4c4': 'SM3TJ7J6DR9KFQXQGK9SY7M7AEJT33C8NV639J99C',
-            'Reserved_for_exchange_partner_1': 'SM17BKV3759FG8H6CSZP2NNE2SPV34MZD0YHHA6AT',
-            'Reserved_for_exchange_partner_2': 'SM23PYTSHF5EP1QJA0VVDEZPXCMJXFQQ2PF8W49BW',
+            'Reserved_for_exchange_partner_1': 'SP2660ENGJMKZ2K8AT7EEPAJTEAHXF7RYB7C1S82G',
+            'Reserved_for_exchange_partner_2': 'SP2660ENGJMKZ2K8AT7EEPAJTEAHXF7RYB7C1S82G',
```
Comment (zone117x) [inline blockstack/blockstackd.py:3808]: @jessesoslow can you confirm these are the correct addresses?

Comment (jessesoslow) [reply blockstack/blockstackd.py:3808]: @zone117x Confirmed that these are correct.

Can you confirm that using the same address twice here isn't going to cause an issue? cc: @kantai

Comment (zone117x) [reply blockstack/blockstackd.py:3808]: Already tested with a re-export, rebuild, and query against `http://localhost:20443/v2/accounts/ST2660ENGJMKZ2K8AT7EEPAJTEAHXF7RYB6W8WZ5W` (the testnet form of the address).

```json
{
  "balance": "0x0000000000000000000003081a211f40",
  "locked": "0x00000000000000000000000000000000",
  "unlock_height": 0,
  "nonce": 0
}
```

`0x0000000000000000000003081a211f40` == `2500000000000 + 833333000000`

Comment (zone117x) [reply blockstack/blockstackd.py:3808]: An earlier PR updated the genesis account credit function to support multiple addresses: https://github.com/blockstack/stacks-blockchain/blob/465f70cd9d80be2caeddcd0ea7369abcd32c1e51/src/chainstate/stacks/db/accounts.rs#L246-L256
@kantai if you want to take a look ^

**Review by kantai — APPROVED** [review]
LGTM!

**Pull request merged**